components/Buttons/SkipButton.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export default function SkipButton({
+  setButtonTextState,
+  setDialogState,
+  buttonTextValue,
+  dialogValue,
+}) {
+  function skipButton() {
+    setButtonTextState(buttonTextValue);
+    setDialogState(dialogValue);
+  }
+
+  return (
+    <SkipText onClick={skipButton}>
+      I am a very boring person and do not want to read all this text.
+    </SkipText>
+  );
+}
+
+const SkipText = styled.button`
+  background-color: var(--color8);
+  color: var(--color6);
+  position: fixed;
+  bottom: 11vh;
+  left: 50vw;
+  width: 80vw;
```

Comment:
I think the button could have a more responsive size with "fit-content" :) 
Also, usually you don't want to encourage the user to skip the instructions, so maybe the button could be less ..red to call less attention to it :)